docs/installation.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+The prfmodel package is currently only available as a development version that must be installed from GitHub.
+Please note that this version is still under active development and subject to constant change.
+Installing prfmodel requires Python version >= 3.10 and <= 3.12.
+
+## Installing the development version
+
+We recommend installing prfmodel in a virtual environment to prevent conflicts with other packages,
+for example using the [venv](https://docs.python.org/3/library/venv.html) package.
+
+On Linux/MacOS (using bash/zsh):
+
+```bash
+python -m venv my_venv
+source my_venv/bin/activate # Activate the virtual environment
+```
+
+On Windows (using cmd.exe):
+
+```console
+python -m venv my_venv
+my_venv\Scripts\activate.bat # Activate the virtual environment
+```
+
+To install the development version of prfmodel from GitHub:
+
+```bash
+git clone git@github.com:popylar-org/prfmodel.git
+cd prfmodel
+python -m pip install .
+```
```

Comment:
I think it might be better to reverse the order, ie first clone, then `cd` into the repo, then create the venv? The way it's currently described, users may end up with their venv not in the repo root. 
I also noted that using a venv is not mentioned in the README; for consistency, perhaps you can add that there too